addon/components/paper-autocomplete.js

```diff
@@ -2,326 +2,83 @@
-  }),
-
-  lookupLabelOfItem(model) {
-    return this.get('lookupKey') ? get(model, this.get('lookupKey')) : model;
   },
 
   actions: {
-    clear() {
-      this.set('searchText', '');
-      this.set('selectedIndex', -1);
-      this.set('model', null);
-      this.set('hidden', this.get('shouldHide'));
-    },
 
-    pickModel(model) {
-      this.set('model', model);
-      let value = this.lookupLabelOfItem(model);
-      // First set previousSearchText then searchText ( do not trigger observer only update value! ).
-      this.set('previousSearchText', value);
-      this.set('searchText', value);
-      this.set('hidden', true);
+    onFocus(event) {
+      this.send('activate');
+      this.publicAPI.actions.open(event);
+      let action = this.get('onfocus');
+      if (action) {
+        action(this.publicAPI, event);
+      }
     },
 
-    inputFocusOut() {
-      this.set('hasFocus', false);
-      if (this.get('noBlur') === false) {
-        this.set('hidden', true);
+    onBlur(event) {
+      this.send('deactivate');
+      let action = this.get('onblur');
+      if (action) {
+        action(this.publicAPI, event);
       }
     },
 
-    inputFocusIn() {
-      this.set('hasFocus', true);
-      this.set('hidden', this.get('shouldHide'));
+    onCreate(text) {
+      if (this.get('onCreate')) {
+        this.get('onCreate')(text);
+      }
+      this.publicAPI.actions.close();
     },
 
-    inputKeyDown(value, event) {
-      switch (event.keyCode) {
-        case this.get('constants').KEYCODE.DOWN_ARROW:
-          if (this.get('loading')) {
-            return;
-          }
-          this.set('selectedIndex', Math.min(this.get('selectedIndex') + 1, this.get('suggestions').length - 1));
-          break;
-        case this.get('constants').KEYCODE.UP_ARROW:
-          if (this.get('loading')) {
-            return;
-          }
-          this.set('selectedIndex', this.get('selectedIndex') < 0 ? this.get('suggestions').length - 1 : Math.max(0, this.get('selectedIndex') - 1));
-          break;
-        case this.get('constants').KEYCODE.TAB:
-        case this.get('constants').KEYCODE.ENTER:
-          if (this.get('hidden') || this.get('loading') || this.get('selectedIndex') < 0 || this.get('suggestions').length < 1) {
-            return;
-          }
-          this.send('pickModel', this.get('suggestions').objectAt(this.get('selectedIndex')));
-          break;
-        case this.get('constants').KEYCODE.ESCAPE:
-          this.set('searchText', '');
-          this.set('selectedIndex', this.get('defaultIndex'));
-          this.set('model', null);
-          this.set('hidden', this.get('shouldHide'));
-          break;
-        default:
-          break;
+    scrollTo(option /*, e */) {
+      if (!self.document || !option) {
+        return;
       }
-    },
+      let publicAPI = this.get('publicAPI');
+      let optionsList = self.document.getElementById(`ember-power-select-options-${publicAPI.uniqueId}`);
```

Comment:
Understood. When I pointed my existing project to the latest commit f780b54 I got those 2 lines as lint errors.